libraries/botbuilder-dialogs-adaptive-teams/src/actions/sendMEAuthResponse.ts

```diff
@@ -7,13 +7,13 @@
  */
 
 import { Activity, CardAction, MessagingExtensionResult } from 'botbuilder';
+import { BaseAuthResponseDialog } from './baseAuthResponseDialog';
 import { DialogContext } from 'botbuilder-dialogs';
-import { BaseAuthResponseDialog, BaseAuthResponseDialogConfiguration } from './baseAuthResponseDialog';
 
 /**
  * Send a messaging extension 'result' response when a Teams Invoke Activity is received with activity.name='composeExtension/queryLink'.
  */
-export class SendMEAuthResponse extends BaseAuthResponseDialog implements BaseAuthResponseDialogConfiguration {
```

Comment:
No extra configuration, so `implements T` is implied by inheriting from base classs.